pages/inverters/[uuid].tsx

```diff
@@ -55,3 +55,5 @@ export const getServerSideProps = withSites({
 });
 
 export default Inverters;
+
+(Inverters as any).hideNav = true;
```

Comment:
These casts to `any` don't seem to be necessary for building/linting the app on my setup at least. Were you running into a problem that required adding them? I think they can be removed otherwise.

It is a bit strange that you can get away without a cast to `any` in a case like this, but I think it's actually a builtin TypeScript case specifically for React that allows for this.